buildSrc/src/main/groovy/checkstyle-conventions.gradle

```diff
@@ -12,6 +12,12 @@ dependencies {
     checkstyle libraries["checkstyleSevntu"]
 }
 
+configurations.checkstyle {
```

Comment:
just a comment needed to explain what/why this is doing before ready for merging